dts/bindings/iio/adc/nuvoton,npcx-adc.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX-ADC node
+
+compatible: "nuvoton,npcx-adc"
+
+include: [adc-controller.yaml]
+
+properties:
+    reg:
+        required: true
+    clocks:
+        required: true
+    label:
+        required: true
+    pinctrl:
```

Comment:
this should be `pinctrl-0`

Reply:
No only adc driver uses "pinctrl" property but also all the other npcx drivers did. If we really want to change the property name, I suggest create a PR to rename it in all npcx drivers. Might I know what's the number means after dash? Thanks.

Reply:
Please refer the [PR29384](https://github.com/zephyrproject-rtos/zephyr/pull/29384). Since the changes is not adc driver only, I collect all changes in this PR. And will add related change for adc later.

Reply:
Done.